kt/godot-library/src/main/kotlin/godot/core/Vector2i.kt

```diff
@@ -101,6 +96,29 @@ class Vector2i(
      */
     fun minAxisIndex() = if (x < y) Vector2.AXIS_X else Vector2.AXIS_Y
 
+    /**
+     * Returns the vector with each component set to one or negative one, depending on the signs of the components.
+     */
+    fun sign() = Vector2i(kotlin.math.sign(x.toFloat()), kotlin.math.sign(y.toFloat()))
+
+    /**
+     * Returns a new vector with each component snapped to the closest multiple of the corresponding component in [step].
+     */
+    fun snapped(step: Vector2i) = Vector2i(
+        snapped(x, step.x),
+        snapped(y, step.y)
+    )
+
+    fun mod(other: Vector2i) = Vector2i(
```

Comment:
It seems that VectorX got plenty new operators including the < operator. Should we add the Comparable interface to them ?